src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -10,9 +12,12 @@ import PaymentMethodActionCreator from '../../payment-method-action-creator';
 import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
 import PaymentStrategyActionCreator from '../../payment-strategy-action-creator';
 import { AdyenPaymentMethodType } from '../adyenv2';
+import { BraintreeGooglePayThreeDSecure, BraintreeVerifyPayload } from '../braintree';
```

Comment:
I think that you can import `BraintreeSDKCreator` from `'../braintree'`. And remove line 16 and 17

```suggestion
import { BraintreeGooglePayThreeDSecure, BraintreeSDKCreator, BraintreeVerifyPayload } from '../braintree';
```

Reply:
Fixed

Reply:
fixed